app/controllers/manage_consents_controller.rb

```diff
@@ -28,6 +28,7 @@ def create
   end
 
   def show
+    current_step # Set the current_step for the back link
```

Comment:
This is a bit awkward, must be a nicer way to generate `@current_step` in the case of the `show` action...